source/qx/io/jsonrpc/message/Request.js

```diff
@@ -1,62 +0,0 @@
-qx.Class.define("qx.io.jsonrpc.message.Request",{
-  extend: qx.io.jsonrpc.Notification,
-  statics: {
-    /**
-     * Static counter for all request ids
-     */
-    __current_request_id : 0,
-
-    getCurrentId() {
-      return qx.io.jsonrpc.message.Request.__current_request_id;
-    },
-
-    resetId() {
```

Comment:
I need it to get predictable results during testing. Since it is a static method of the class, I think it's ok. The counter is manually accessible anyways.... It would only be different if the counter was a real private variable that needed to be protected.